### PR TITLE
Create Handling Events with Stencil

### PR DESCRIPTION
**Pull request by Pawan85**
I am Pawan. Please approve it.

### CODE REVIEW

File: Handling Events with Stencil [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import  { Component, Prop, State, h } from  '@stencil/core';
+
+@Component({
+   tag: 'my-rating',
+   styleUrl: 'my-rating-component.css',
```
Comment (isnur) [inline Handling Events with Stencil:5]: where is the css file? @Pawan85

Comment (Pawan85) [reply Handling Events with Stencil:5]: <!DOCTYPE html>
<html dir="ltr" lang="en">
<head>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=5.0">
  <title>Stencil Component Starter</title>
  <script type="module" src="/build/rating-stencil-component.esm.js"></script>
  <script nomodule src="/build/rating-stencil-component.js"></script>
  </head>
  <body>
    <my-rating id="myRatingComponent" max-value="6" value="2"></my-rating>
  </body>
  </html>

**Comment by Pawan85** [comment]
<!DOCTYPE html>
<html dir="ltr" lang="en">
<head>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=5.0">
  <title>Stencil Component Starter</title>
  <script type="module" src="/build/rating-stencil-component.esm.js"></script>
  <script nomodule src="/build/rating-stencil-component.js"></script>
  </head>
  <body>
    <my-rating id="myRatingComponent" max-value="6" value="2"></my-rating>
  </body>
  </html>

**Comment by Pawan85** [comment]
I have provided css code

**Comment by Pawan85** [comment]
Hacktoberfest accepted and merge.

**Pull request closed without merging**